src/ui/screens/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -159,7 +159,7 @@ VPNViewBase {
                     VPNSubscriptionData.planCurrency,
                     VPNSubscriptionData.planAmount,
                 ),
-                type: "text",
+                type: (Qt.locale().name === "en_US" || Qt.locale().name === "en_CA" ) && VPNFeatureList.get("bundleUpgrade").isSupported ? "text-upgrade" : "text",
```

Comment:
Question: Checking on the locale is only a temporary solution?

Reply:
semi-permanent - we're limiting this to various "en" locales for now (and unfortunately). this stipulation is based on geographic availability of the relay phone masking feature.